pkg/codegen/testing/test/testdata/regress-py-12980/python/pulumi_myPkg/mymod/childb/member_b2.py

```diff
@@ -0,0 +1,116 @@
+# coding=utf-8
+# *** WARNING: this file was generated by test. ***
+# *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import copy
+import warnings
+import pulumi
+import pulumi.runtime
+from typing import Any, Mapping, Optional, Sequence, Union, overload
+from ... import _utilities
+from . import outputs
+from ... import mymod
+from ... import mymod as _mymod
```

Comment:
Not sure if it's necessarily related to this change, but these two lines are quite odd to have. I would think one of those would be sufficient. Enums seem to be referenced with `mymod` below. Resources/Types with `_mymod`.

Reply:
I updated the code to use the unprefixed name for all imports.

Reply:
I'll back this change out again since it requires updating some test snapshots and is not related to the original fix. I prefer making that change in a separate PR to keep things separate.